map/src/components/resultDetail.tsx

```diff
@@ -0,0 +1,247 @@
+        </a>
+      )),
+    );
+  }
+  if (info.email) {
+    items.push(
+      ...info.email.map(email => (
+        <a
+          key={items.length}
+          href={`mailto:${email}`}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <MdEmail />
+          {email}
+        </a>
+      )),
+    );
+  }
+  if (info.facebookGroup) {
+    items.push(
+      <a
+        key={items.length}
+        href={info.facebookGroup}
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <MdLanguage />
+        {t(lang, s => s.results.contact.facebookGroup)}
+      </a>,
+    );
+  }
+  if (info.web) {
+    items.push(
+      ...Object.entries(info.web).map(entry => (
+        <a
+          key={items.length}
+          href={entry[1]}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <MdLanguage />
+          {entry[0]}
+        </a>
+      )),
+    );
+  }
+  if (items.length === 0) {
+    return null;
+  }
+  return (
+    <div className="contact-group">
+      <strong>{label}</strong>
+      <ul>
+        {items.map((item, i) => (
+          <li key={i}>{item}</li>
+        ))}
+      </ul>
+    </div>
+  );
+};
+
+const ResultDetail = ({ result, lang, fullDetail = false }: Props) => {
+  const selectedResultSource =
+    result?.info.source && SOURCES[result.info.source.name];
+  const selectedResultSentenceType =
+    result?.info.type.type === 'mutual-aid-group' ||
+    result?.info.type.type === 'org'
+      ? result.info.type.type
+      : 'project';
+  return (
+    <Div>
+      <div className="disclaimer">
+        <strong>{t(lang, s => s.results.details.disclaimer.header)}</strong>
+        <br />
+        {t(lang, s => s.results.details.disclaimer.content)}
+      </div>
+
+      {fullDetail && <div className="name">{result.info.contentTitle}</div>}
+      {fullDetail && result.info.loc.description && (
+        <div className="location">{result.info.loc.description}</div>
+      )}
+      {fullDetail && (
+        <MarkerType className="marker-type" type={result.info.type} />
+      )}
+      {selectedResultSource && (
+        <div className="source">
+          {t(lang, s => s.results.details.source, {
+            type: key => (
+              <span key={key}>
+                {t(lang, s => s.markerTypeSentence[selectedResultSentenceType])}
+              </span>
+            ),
+            source: key => (
+              <a
+                key={key}
+                href={selectedResultSource.href}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                {selectedResultSource.label}
+              </a>
+            ),
+          })}
+        </div>
+      )}
+      {result.info.contentBody && (
+        <div>
+          <div className="content">{result.info.contentBody}</div>
+        </div>
+      )}
+      {result.info.contact && (
+        <div>
+          {contactInfo(
```

Comment:
If you want to write this as a `map`, you can use `Object.keys` to access all the keys as an array 😄 it's ok to leave this in my opinion since there are only 3, but just saying since I'm not sure if you're aware of `Object.keys`

Reply:
haha thats true

Reply:
will leave this one out  for next PR too, it was part of old code